src/dataframe/dataframe.jl

```diff
@@ -1024,15 +930,23 @@ select!(df::DataFrame, c::Any) = select!(df, index(df)[c])
 
 """
     select(df::DataFrame, inds, copycols::Bool=true)
+    select(dfv::SubDataFrame, inds, copycols::Bool=true)
+    select(dfr::DataFrameRow, inds)
 
-Create a new `DataFrame` that contains columns from `df` specified by `inds` and return it.
-
-Argument `inds` can be any index that is allowed for column indexing of a `DataFrame`.
-
+If `df` is a `DataFrame` create a new `DataFrame` that contains columns from `df`
```

Comment:
```suggestion
If `df` is a `DataFrame` return a new `DataFrame` that contains columns from `df`
```
(and then remove "and return it")